config/snuba/settings.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+env = os.environ.get
+
+DEBUG = True
+
+DEFAULT_RETENTION_DAYS = env("SENTRY_EVENT_RETENTION_DAYS", 90)
+
+REDIS_HOST = env("REDIS_HOST", "localhost")
+REDIS_PORT = int(env("REDIS_PORT", 6379))
+REDIS_PASSWORD = env("REDIS_PASSWORD")
+REDIS_DB = int(env("REDIS_DB", 1))
+USE_REDIS_CLUSTER = False
+
+# Dogstatsd Options
+DOGSTATSD_HOST = env("DOGSTATSD_HOST")
+DOGSTATSD_PORT = env("DOGSTATSD_PORT")
+
+SENTRY_DSN = env("SENTRY_DSN")
+
+ERRORS_ROLLOUT_ALL = False
+ERRORS_ROLLOUT_WRITABLE_STORAGE = False
```

Comment:
I was thinking if we can make this simpler.
How do you plan to roll this out on the development environment? Is it going to be by referrer there as well, or are you planning for a quick cutover of these flags?
If you are doing the second, why not tying those parameters to environment variables like the connection ones and setting the variable in the docker settings as we are doing for `CLICKHOUSE_HOST` without having a full config file ?

Anyway I would not block on this as this is tepmorary.

Reply:
It's an immediate cutover as soon as the migration is in. The old table will be dropped completely dropped by the time these are flipped to true.

I personally dislike the practice of making everything in settings an environment variable then attempting to cast it to the right type as a way to configure settings in sentry. I think this is a more flexible solution that allows us to configure any other part of settings exclusively for this environment in the future without making everything into an env variable.

Reply:
The downside of the whole file here is that we will have to keep in mind another file to verify when we change config structure. We have control of that file so it is not a huge issue, just a bit annoying. I would also remove it once the rollout is done as it will become again the same as the the docker one.

I see your point about environment variables.

Reply:
I would leave this file in permanently. IMO Having onpremise and Devservices share the same config file then adding tons of environment variables to get around the fact that we don't want them to actually be the same in some cases is something we should just avoid.